src/main/java/BasisSupport/transformString.java

```diff
@@ -0,0 +1,93 @@
+package BasisSupport;
+import Support.Time;
+
+// Transform the line with "/" to common
+public class transformString {
+    public static String transformString(String line) {
+        String transformedLine;
+        String[] wordsListOfLine = line.split(" ");
+        boolean firstTimeOccurence = true; // Indicates the occurrence time of "/"
+
+
+        // In order to provide brackets and colon to the content
+        if (line.contains("/")) {
+            for (int i = 0; i < wordsListOfLine.length; i++) {
+                if (wordsListOfLine[i].startsWith("/")) {
+                    if (firstTimeOccurence) {
+                        wordsListOfLine[i] = "(" + wordsListOfLine[i].substring(1) + ":";
+                        firstTimeOccurence = false;
+                    } else {
+                        wordsListOfLine[i] = wordsListOfLine[i].substring(1) + ":";
+                    }
+                }
+            }
+            transformedLine = String.join(" ", wordsListOfLine) + ")";
+        } else {
+            transformedLine = line;
+        }
```

Comment:
Consider abstracting the logic containing the if-else statements to minimize deeply-nested code blocks and to increase readability.